proposals/drafts/campaign_finance_filings.rst

```diff
@@ -0,0 +1,425 @@
+-------------------------------
+
+id
+    Open Civic Data-style id in the format ``ocd-cf-committeestatusupdate/{{uuid}}``
+
+new_status
+    New status to set for Committee. This could be an enumerated type or a
+    free-text field.
+
+description
+    String containing whatever associated text we got along with the status
+    change.
+
+CommitteeAttributeUpdate (Section)
+----------------------------------
+
+id
+    Open Civic Data-style id in the format ``ocd-cf-committeeattributeupdate/{{uuid}}``
+
+attribute_to_update
+    Attribute in the Committee object to change.
+
+new_attribute_value
+    Value to set for the attribute in the Committee object.
+
+Candidate
+---------
+
+This is not necessarily a Person: for instance, "yes" or "no" to a ballot
+measure.
+
+id
+    Open Civic Data-style id in the format ``ocd-cf-candidate/{{uuid}}``
+
+election
+    Election this Candidate is contesting.
+
+party
+    **optional**
+    Party this Candidate is aligned with.
+
+office
+    Office for which this Candidate is running.
+
+regulator
+    **repeated**
+    **optional**
+    Any applicable Regulators who have jurisdiction over this Candidate. For
+    example, FEC and PDC have jurisdiction over different candidates in WA
+    on a given election day. I suppose it's possible some Candidates won't have
+    Regulators (God help us all).
+
+Jurisdiction
+------------
+
+id
+    Open Civic Data-style id in the format ``ocd-cf-jurisdiction/{{uuid}}``
+
+name
+    Name to use for this Jurisdiction.
+
+geography
+    **optional**
+    Geography object containing the boundaries of this Jurisdiction. Reluctantly
+    optional.
+
+government_level
+    **repeated**
+    Government Level with...jurisdiction over this Jurisdiction.
+
+Office
+------
+
+id
+    Open Civic Data-style id in the format ``ocd-cf-office/{{uuid}}``
+
+name
+    Name of the Office.
+
+government_level
+    Government Level that this Office belongs to.
+
+Party
+-----
+
+id
+    Open Civic Data-style id in the format ``ocd-cf-party/{{uuid}}``
+
+name
+    Name of the Party.
+
+Regulator
+---------
+
+id
+    Open Civic Data-style id in the format ``ocd-cf-regulator/{{uuid}}``
+
+name
+    Name of the Regulator.
+
+url
+    **optional**
+    URL associated with the Regulator; this should probably either not be part
+    of the spec, or associated with a much more rigorous system of URL prefixes
+    for different report types, etc.
+
+government_level
+    Government Level that this Regulator belongs to.
+
+Election
+--------
+
+id
+    Open Civic Data-style id in the format ``ocd-cf-election/{{uuid}}``
+
+date
+    Date of the Election.
+
+jurisdiction
+    Jurisdiction with at least one race occurring on this Election date.
+
+is_primary
+    Whether this Election is a primary race or not for this Jurisdiction.
+
+primary_parties_involved
+    **repeated**
+    **optional**
+    If this is a primary, each Party involved in this Election.
+
+GovernmentLevel
```

Comment:
When's an example of a case where you know the jurisdiction but you still want to know the "goverment level"?